WireUI/Package.swift

```diff
@@ -38,6 +40,16 @@ let package = Package(
 
         .target(name: "WireFolderPickerUI", dependencies: ["WireReusableUIComponents"]),
 
+        .target(
+            name: "WireIndividualToTeamMigrationUI",
+            dependencies: [
+                .product(name: "WireDomainAPI", package: "WireDomainPackage"),
+                "WireFoundation",
+                "WireReusableUIComponents"
+            ]
```

Comment:
Please use the `SwiftGenPlug` for creating string constants.
```suggestion
            ],
            plugins: [
                .plugin(name: "SwiftGenPlugin", package: "WirePlugins")
            ]
```

Then copy `./WireUI/Sources/WireSidebarUI/.swiftgen.yml` to `WireUI/Sources/WireIndividualToTeamMigrationUI/.swiftgen.yml`.

Reply:
Actually, now that you have used string catalogs, this won't work.
We had decided to not continue with string catalogs, but we should have told you.

Reply:
Ah! Yes, that would have been good to know. Well, it will have to be done later.

Reply:
not a blocker for now